encoding/encodeInto.any.js

```diff
@@ -1,3 +1,13 @@
+function createBuffer(type, length = 0) {
```

Comment:
It may be worth pulling this out into /common

Reply:
I was thinking to maybe do that, okay. And then use some closure to only do the WebAssembly thing once and hide the constructor. `/common/sab.js` okay? I'll prolly clean up these 4 PRs with that tomorrow then.

Reply:
SGTM